sdcm/microbenchmarking.py

```diff
@@ -97,7 +97,12 @@ def get_commit_id(x):
             last_val = get_metrica_val(list_of_results_from_db[last_idx])
             last_commit = get_commit_id(list_of_results_from_db[last_idx])
             cur_val = float(current_result["results"]["stats"][metrica])
-            min_result = min(list_of_results_from_db, key=get_metrica_val)
+
+            if 'frag/s' in metrica:
```

Comment:
I don't like this condition and the one that check diff.
we should define list of metrics that are "higher_better" and list of metrics that are "lower_better".
Then, per metric type we check accordingly.

For now we stay only with "frag/s" and "aio", but in the future it may be changed again.

frag/s is "higher better".
aio I need to verify.

Reply:
Implement according your comments. I think  it could refactoring for better solution.
Now it is in common style of current implementation.